config.py

```
@@ -17,6 +17,9 @@
 
 
 # ============= 其他配置 =============
+# 测试模式
+TEST_MODE = False  # 设置为True以启用测试模式（仅处理前几个视频以加快测试速度）
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)